pages/account/statistics.js

```diff
@@ -272,7 +272,7 @@ export default function Statistics({ data, profile, progress, BASE_URL }) {
             {data.links &&
               data.links.individual.map((link) => (
                 <tr key={link.url}>
-                  <td className="md:whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium text-primary-high dark:text-primary-low sm:pl-6">
+                  <td className=" truncate break-words max-sm:max-w-[200px] max-w-lg bg-dark py-4 pl-4 pr-3 text-sm font-medium text-primary-high dark:text-primary-low">
```

Comment:
Is the px width needed? Seems fragile, it should automatically truncate on overflow

Reply:
it doesn't interfere with any modern devices as 99.99999999% devices are above width 320px. I'm just making sure that there is atleast some breathing area is given to the link inorder to make it readable. I don't think it is a problem.

Reply:
px measurement is not great, if not needed most of the time best to remove it - we don't need to over optimise

Reply:
![image](https://github.com/EddieHubCommunity/BioDrop/assets/53367382/e003d2d1-4fc2-4b4e-9b36-55d1fad2802e)
 
For some strange reason the table is overflowing after removing the breakpoint.